composio/sdk/sdk.py

```diff
@@ -312,84 +313,60 @@ def get_app(self, app_name: str):
 
     def get_list_of_actions(
         self,
-        apps: list[App] = None,
-        use_case: str = None,
-        actions: list[Action] = None,
-        tags: list[Union[str, Tag]] = None,
-        limit: int = None,
+        apps: list[App] = [],
+        actions: list[Action] = [],
+        use_case: str = "",
+        tags: list[Union[str, Tag]] = [],
+        limit: int = 0,
     ) -> list:
```

Comment:
Are we missing empty apps case here -> get all apps and their actions?

Reply:
Solved.